test/parallel/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,13 +15,11 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[22, 21, 20, 19, 18, 17, 16, 16, 17, 18, 19, 20, 21, 22].forEach((exponent) => {
+[22, 16].forEach((exponent) => {
```

Comment:
> Depends on whether or not you want the test to fail reliably on all (supported) platforms when there's a problem.

This test used to be an expected-fail. This PR moved it into the normal test section as it should be passing/deterministic.